modules/sdk-coin-doge/src/doge.ts

```diff
@@ -78,4 +80,16 @@ export class Doge extends AbstractUtxoCoin {
   ): Promise<CrossChainRecoverySigned<TNumber> | CrossChainRecoveryUnsigned<TNumber>> {
     return super.recoverFromWrongChain(params);
   }
+
+  parseUnspents<TNumber extends number | bigint>(unspents: Unspent<TNumber>[]): Unspent<TNumber>[] {
+    return unspents?.map((unspent) => {
+      const newUnspent: Unspent<TNumber> = Object.assign(
+        {},
+        unspent.id,
+        unspent.address,
+        utxolib.bitgo.toTNumber((unspent as any)?.valueString, 'bigint')
+      );
```

Comment:
`Object.assign` will assign the _properties_ of the arguments, not the arguments themselves (it cannot know the property name)

in modern JS you can use the spread syntax anyway

```suggestion
      return { ...unspent, value: BigInt(unspent.valueString); }
```

Reply:
done